components/board/upboard/data.go

```diff
@@ -0,0 +1,60 @@
+package upboard
+
+import "go.viam.com/rdk/components/board/genericlinux"
+
+const upboard = "up_4000"
+
+var boardInfoMappings = map[string]genericlinux.BoardInformation{
+	upboard: {
+		[]genericlinux.PinDefinition{
+			/*
+				pinout for up4000: https://github.com/up-board/up-community/wiki/Pinout_UP4000
+				GPIOChipRelativeIDs: {ngpio : base-linux_gpio_number}
+				GPIOChipSysFSDir: path to the directory of a chip. Can be found from the output of gpiodetect
+			*/
+			// GPIO pin definition
+			{map[int]int{78: 73}, map[int]string{}, "INT3452:01", 29, 0, "GPIO10", "", "", -1},
+			{map[int]int{77: 46}, map[int]string{}, "INT3452:01", 31, 0, "BCM26", "", "", -1},
+
+			{map[int]int{77: 48}, map[int]string{}, "INT3452:01", 18, 0, "BCM24", "", "", -1},
+			{map[int]int{77: 45}, map[int]string{}, "INT3452:01", 22, 0, "BCM25", "", "", -1},
```

Comment:
I can't figure out the order on these. Please sort them by pin number (e.g., this line should be 22nd), or else explain what the current ordering is.

Reply:
Please ignore the pin definition codes for now. This is going to change a lot in the near future.